data/appstream/meson.build

```diff
@@ -0,0 +1,8 @@
+i18n.merge_file(
+  'gpaste-appdata',
```

Comment:
Is this param used for something? (I'm not familiar with meson yet)

Reply:
It's simply the name of the custom target. Meson will display it during the build process, something along the lines of `generating custom target gpaste-appdata`